examples/platforms/cc2538/random.c

```diff
@@ -106,7 +106,7 @@ ThreadError otPlatRandomSecureGet(uint16_t aInputLength, uint8_t *aOutput, uint1
     if (otPlatRadioIsEnabled(sInstance))
     {
         channel = 11 + (HWREG(RFCORE_XREG_FREQCTRL) - 11) / 5;
-        otPlatRadioSleep(sInstance);
+        otPlatRadioSleep(sInstance); // BUGBUG - This can fail if currently transmitting. What then?
```

Comment:
Yeah, I'm not really a fan of how the `otPlatRadio` API turned out. Its state machine is rather fragile for my tastes.

I think the most simple solution here would be to ensure that `otPlatRadioSleep()` never fails.